frontend/src/reducer/currentWorkspace.js

```diff
@@ -97,37 +94,37 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         sidebarEntryList: action.workspaceDetail.sidebar_entries.map(sbe => serialize(sbe, serializeSidebarEntryProps))
       }
 
-    case `${SET}/${WORKSPACE_MEMBER_LIST}`:
+    case `${SET}/${USER_ROLE_LIST}`:
```

Comment:
add comment explaining that USER_ROLE is an object made from an user and a role

Reply:
done